changes/2113.misc.rst

```diff
@@ -0,0 +1 @@
+Integration tests in CI now run on arm64 Linux.
```

Comment:
This should be promoted to feature - "Linux on ARM is now a fully supported platform".

Reply:
That's fair; I'll call that out - although, I think I should say arm64 since things like Pi 2/3 aren't covered by this testing.